cumulus/parachains/runtimes/assets/asset-hub-rococo/src/genesis_config_presets.rs

```diff
@@ -109,43 +110,21 @@ pub fn get_preset(id: &PresetId) -> Option<Vec<u8>> {
 		Ok(sp_genesis_builder::LOCAL_TESTNET_RUNTIME_PRESET) => asset_hub_rococo_genesis(
 			// initial collators.
 			vec![
-				(
-					get_account_id_from_seed::<sr25519::Public>("Alice"),
-					get_collator_keys_from_seed::<AuraId>("Alice"),
-				),
-				(
-					get_account_id_from_seed::<sr25519::Public>("Bob"),
-					get_collator_keys_from_seed::<AuraId>("Bob"),
-				),
+				(Sr25519Keyring::Alice.to_account_id(), Sr25519Keyring::Alice.public().into()),
+				(Sr25519Keyring::Bob.to_account_id(), Sr25519Keyring::Bob.public().into()),
 			],
-			vec![
-				get_account_id_from_seed::<sr25519::Public>("Alice"),
-				get_account_id_from_seed::<sr25519::Public>("Bob"),
-				get_account_id_from_seed::<sr25519::Public>("Charlie"),
-				get_account_id_from_seed::<sr25519::Public>("Dave"),
-				get_account_id_from_seed::<sr25519::Public>("Eve"),
-				get_account_id_from_seed::<sr25519::Public>("Ferdie"),
-				get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-				get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-				get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
-				get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
-				get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
-				get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
-			],
-			ROC * 1_000_000,
+			Sr25519Keyring::iter().take(12).map(|x| x.to_account_id()).collect(),
```

Comment:
I noticed that in several places we take the first 12 (i.e. we skip ONE and TWO) for the "well known" keys and 6 for the "invulnerables".

Maybe is worth define in `sp-keyring` a couple of constants? Something like `WELL_KNOWN_KEYS_COUNT = 12` and `INVULNERABLE_KEYS_COUNT = 6` and use the constants instead of magic numbers?

Or **even better** define for the `Keyring` enum two methods `well_known()` and `invulnerable()` which internally just call `Self::iter().take(12)` and `Self::iter().take(6)` respectivelly